src/SDKs/CognitiveServices/dataPlane/Vision/CustomVision/Training.Tests/Microsoft.Azure.CognitiveServices.Vision.CustomVision.Training.Tests.csproj

```diff
@@ -10,6 +10,10 @@
     <TargetFramework>netcoreapp1.1</TargetFramework>
   </PropertyGroup>
   
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netcoreapp2.0|AnyCPU'">
```

Comment:
1. remove all conditions for the supressions
2. Add coimments regatrding why supressions are being added